plugins/lookup/github_app_access_token.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023, Poh Wei Sheng <weisheng-p@hotmail.sg>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: github_app_access_token
+    author:
+      - Poh Wei Sheng (@weisheng-p)
+    short_description: This plugin generates Github App Access token
+    version_added: '3.1.0'
+    requirements:
+      - jwt (https://github.com/GehirnInc/python-jwt)
+    description:
+      - This generates a Github access token that can be used with a git command, if you use a Github App.
+    options:
+      key_path:
+        description:
+        - Path to your private key
+        required: true
+        type: str
+      app_id:
+        description:
+        - Your Github App Id, you can find this in the settings page
```

Comment:
GitHub has a capital H. ID should always be fully capitalized (https://docs.ansible.com/ansible/devel/dev_guide/style_guide/spelling_word_choice.html#acronyms - ID is common enough that it doesn't need to be written out first IMO).
```suggestion
        - Your GitHub App ID, you can find this in the settings page.
```